docs/settings/reporting-settings.asciidoc

```diff
@@ -229,10 +229,10 @@ a| `xpack.reporting.capture.browser`
   See OWASP: https://www.owasp.org/index.php/CSV_Injection
   Defaults to `true`.
 
-| `xpack.reporting.csv.enablePanelActionDownload`
-  | Enables CSV export from a saved search on a dashboard. This action is available in the dashboard
-  panel menu for the saved search.
-  Defaults to `true`.
+| `xpack.reporting.csv` `.enablePanelActionDownload`
```

Comment:
@gchaps would you mind taking a look at this change? I added a space character in the label of the setting so that the line can wrap and the table cell would not be forced to be too wide. 

This is in regards to the screenshot, where you can see a horizontal scrollbar in the table of CSV settings (taken in Chrome).